### ISSUE TITLE
Widget `relation`, default value not set, if `multiple` is `true`

### ISSUE THREAD

**Issue by samtrion** [sveltia/sveltia-cms#117]
When I use the following configuration, the `default` value is not taken for new posts. If I set `multiple` to `false`, it works as aspected.

```yml
- label: Authors
  name: authors
  widget: relation
  i18n: duplicate
  collection: data-authors
  value_field: "id"
  search_fields: ["id", "name"]
  display_fields: ["name"]
  multiple: true
  min: 1
  max: 3
  default: "default"
```

**Comment by kyoshino** [comment]
I think, when `multiple: true`, the proper configuration would be `default: ['default']` just like the [list widget](https://decapcms.org/docs/widgets/#list). But it’s not working in the current version of Sveltia. Will fix.

**Comment by kyoshino** [comment]
Just shipped [v0.13.1](https://github.com/sveltia/sveltia-cms/releases/tag/v0.13.1) to address this and other issues 🩹
